modules/common/src/test/java/com/ibm/cloud/cloudant/internal/SdkTimeoutTest.java

```diff
@@ -0,0 +1,189 @@
+/**
+ * © Copyright IBM Corporation 2020. All Rights Reserved.
```

Comment:
The year is wrong here
```suggestion
 * © Copyright IBM Corporation 2021. All Rights Reserved.
```

Reply:
Changed in 87d0bf4